src/psyclone/tests/test_files/gocean1p0/kernel_stencil.f90

```diff
@@ -67,11 +67,13 @@ module kernel_stencil
   !===================================================
 
   !> Compute the mass flux in the x direction at point (i,j)
-  subroutine compute_cu_code(i, j, cu, p, u)
+  subroutine compute_cu_code(i, j, cu, p, u, area)
```

Comment:
This kernel doesn't access the `area` array?

Reply:
No, it doesn't - but the kernel declaration contained an additional argument, and the test case needs the fourth parameter. I've added a comment explaining why area is there as a paramter.

Reply:
Oh yes; the meta-data says it needs the cell area. Don't know why I didn't see that last time around.